README.md

```diff
@@ -44,7 +42,7 @@ Logstash plugin can be used as a publisher in pipeline jobs to send the whole lo
 
 It can be used as a wrapper step to send each log line separately.
 
-Note: when you combine with timestamps step, you should make the timestamps the outer most block. Otherwise you get the timestamps as part of the log lines, basically duplicating the timestamp information.
+Note: when you combine with timestamps step, you should make the timestamps the outermost block. Otherwise you get the timestamps as part of the log lines, basically duplicating the timestamp information.
```

Comment:
is this warning still needed? I would have thought that the remove console note code would handle this?

Reply:
As far as I understand the console note removal it would not help here, but I might be wrong.
Care to try it out?